tiatoolbox/models/dataset/classification.py

```diff
@@ -323,6 +332,7 @@ def __init__(
         self._check_input_integrity(mode="wsi")
 
     def __getitem__(self, idx):
+        """Defines the behaviour when an item is accessed."""
```

Comment:
```suggestion
        """Get an item from the dataset."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/a24a4918731addef5dc1da96c9fe6e396b122a37